.gitignore

```diff
@@ -1,4 +1,11 @@
 **/**.DS_Store
 **/*.csv
+**/*.json
 **/.vscode
 **/.txt
+repopack-output.txt
+
+
+# Added by cargo
```

Comment:
the target and debug folders are already in the `.gitignore` in the parser directory

Reply:
The .gitignore does not need any changes